docs/monitoring/autodiscovery.md

```diff
@@ -90,6 +90,30 @@ In container environments you add the `MONITORING_TOKEN` environment variable, o
 
 In all environments this makes it possible to use different monitoring tokens, and thus different Sematext Apps for the same service type running on the same machine or in the same Kubernetes/Swarm cluster.
 
+### Switching from manual to automatic monitoring
+
+The switch from manual to automatic monitoring is done by first removing manually set up agent and then enabling automatic monitoring in the user interface.
+
+In bare-metal/virtual machine setups there are two types of agents - in-process (with javaagent) and standalone.
+
+If you are using in-process variant, do the following:
```

Comment:
If you are using in-process variant, do the following:
=>
If you are using the in-process type, do the following: